### PR TITLE
Wrap the call to puts Terminal::Table in a rescue block

### PR DESCRIPTION
**Pull request by ohayon**
I fixed a crash just like this last week and it looks like there is another one. This just wraps it in a rescue block so we can get more information about the crash as it happens https://github.com/fastlane/fastlane/pull/9279

### CODE REVIEW

File: fastlane/lib/fastlane/lane_manager.rb [hunk 1]
```diff
@@ -111,13 +111,19 @@ def self.print_table(actions)
...
 
-      puts ""
-      puts Terminal::Table.new(
-        title: "fastlane summary".green,
-        headings: ["Step", "Action", "Time (in s)"],
-        rows: FastlaneCore::PrintTable.transform_output(rows)
-      )
-      puts ""
+      begin
+        require 'terminal-table'
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/lane_manager.rb]: This looks to be a duplicate require from above? ^

File: fastlane/lib/fastlane/lane_manager.rb [hunk 2]
```diff
@@ -111,13 +111,19 @@ def self.print_table(actions)
...
+        puts ""
+        puts Terminal::Table.new(
+          title: "fastlane summary".green,
+          headings: ["Step", "Action", "Time (in s)"],
+          rows: FastlaneCore::PrintTable.transform_output(rows)
+        )
+        puts ""
+      rescue
+        os = Helper.linux? ? 'linux' : 'mac'
```
Comment (joshdholtz) [inline fastlane/lib/fastlane/lane_manager.rb]: Is windows not an option at all?

**Review by joshdholtz — COMMENTED** [review]
@ohayon Two questions for ya ❓ ❓

**Comment by KrauseFx** [comment]
Do we have to potentially add this to every table we print? Is that something we should fix in the table directly instead?

**Comment by lacostej** [comment]
I am looking at #9279, and we now added a `crash` under `print_lane_context`. This means we won't get the original exception printed out, the one from `fastlane/lib/fastlane/lane_manager.rb:58` meaning we might lose some valuable information as to why this happens.

**Comment by KrauseFx** [comment]
Ohh, nice catch @lacostej, we should probably revert #9279 maybe? cc @ohayon

**Comment by ohayon** [comment]
We will only miss the crash if the crash happens due to table printing though right? We can definitely revert it if you think its better, we should look into a better fix for these crashes in the table printing anyways I think.

**Comment by KrauseFx** [comment]
I'd be in favor of reverting to be honest, and try to find out the underlying issue of what's happening. Do I remember correctly that we didn't get a GitHub issue from anyone about this?

**Comment by ohayon** [comment]
Yeah, I would be in favor of reverting it as well. Better to find the root.

**Comment by ohayon** [comment]
Here is a PR to revert the first one of these: https://github.com/fastlane/fastlane/pull/9450

**Pull request closed without merging**